Makefile

```diff
@@ -18,19 +18,30 @@ CURL=curl -sSf
 SOURCE_DIR?=$(dir $(lastword $(MAKEFILE_LIST)))
 SOURCE_DIR:=$(abspath $(SOURCE_DIR))
 ###############################################################################
+# Subcomponent versions:
+CONFD_VER := v0.10.0-scale
+BIRD_VER := v0.2.0
+GOBGPD_VER := v0.1.1
+FELIX_VER := 2.0.0
```

Comment:
The failing test needs felix version `2.0.1-rc1`, `2.0.0` doesn't contain the relevant feature.